lib/utils.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
```

Comment:
I'm not going to block this at the 11th hour (sorry I missed this before), but we should stop making `utils.js` files.  They become a dumping ground of random functions which makes finding functions or browsing the project difficult.  Instead we should try to [organize functions by what they do](https://logdna.blogin.co/posts/99035) (see "No more utils").  This file in particular would benefit from `stdlib`.  We should expose that package to the public so we don't have to duplicate the code here.  When we do that, we can split this file up into something better organized.

Reply:
@darinspivey, yup - this is in my mind, we can do reorganization in the next patch release after this